### DISCUSSION TITLE
Typewriter sound?

### DISCUSSION THREAD

**Discussion by 9acca9** [vkbo/novelWriter#1371] [Ideas]
Well, i really like that. It helps me write, or that is what i think.
If you can add that it will be nice.
Thanks.

(i dont speak english)

**Comment by 9acca9** [comment]
workaround (tested in linux):
https://github.com/aizquier/typewriter-sounds

**Reply by vkbo** [reply]
I don't think this is a feature I would prioritise adding at this point.

**Comment by johnblommers** [comment]
`sudo apt install bucklespring`

Then run `buckle`

Homepage: https://github.com/zevv/bucklespring

This works for sure on Linux. If you're running on proprietary operating system this still might work. RTFM.

Buckle is awesome. Not only are the effects in stereo you get a sound on press and on release. But if you share your writing space with other folks expect to receive annoying stares. Just saying LOL!

**Reply by vkbo** [reply]
Hehe:

> This project emulates the sound of my old faithful IBM Model-M space saver bucklespring keyboard while typing on my notebook, mainly for the purpose of annoying the hell out of my coworkers.

Anyway, I don't think any of these will be high on my ToDo list for the foreseeable future. 😆

**Comment by johnblommers** [comment]
Agreed. You don't really want to add a typewriter sounds program to novelWriter when external tools can provide this feature.

Let me add that I prefer the solution [https://github.com/aizquier/typewriter-sounds](https://github.com/aizquier/typewriter-sounds) over Bucklespring. Typewriter-sounds actually sounds like a real typewriter. Backspace, Shift, carriage return and the subsequent line feed sound of the platen advancing the paper to the next line just sound so freaking good. WooT!

What a delightful distraction LOL!

**Reply by vkbo** [reply]
Just a note to use the reply box when replying to another comment. Linearity is not preserved in GitHub Discussions, so as soon as someone adds a new comment in response to another comment, the threads become impossible to follow.

**Reply by johnblommers** [reply]
Huh! I did not know that. And I've been using GitHub for several years. Thanks for the tip :-)

**Reply by jason-mehmel** [reply]
A quick thank you @johnblommers for re-introducing me to this program and getting me using it!

Focuswrititer has a great typewriter sound option but I'm loving that I can apply this solution to other writing programs!

**Reply by johnblommers** [reply]
@jason-mehmel, you are most welcome. I use [typewriter-sounds](https://github.com/aizquier/typewriter-sounds ) all of the time. It's nice to know others find it helps them get through the day.
